src/components/inputs/CheckboxList.tsx

```diff
@@ -0,0 +1,85 @@
+import { FormEvent, ReactNode } from 'react';
+import { uniqueId } from 'lodash';
+
+export enum LinksPosition {
+  None,
+  Top = 1 << 1,
+  Bottom = 1 << 2,
+  Both = Top | Bottom
+}
+
+type Item = {
+  display: ReactNode
+  value: any
```

Comment:
Later in code, we're assuming this type is `string` (see `Props['value']`). We should either change this type to `string`, or we should change it to `unknown`, and change `Props['value']` to `unknown[]`.